### PR TITLE
fix: search button tooltip and placeholder

### PR DESCRIPTION
**Pull request by yangfei4913438**
### What does it do?

A list page of collectionType objects showing the search button internationalized copy for tooltip end placeholder.

### Why is it needed?

Because the component lacks internationalized copy.

### How to test it?

When the mouse passes over the button, it displays the tooltip, and when the button is clicked, it displays the input box with the placeholder inside.

### CODE REVIEW

File: packages/core/admin/admin/src/content-manager/pages/ListView/index.js [hunk 1]
```diff
@@ -309,7 +309,7 @@ function ListView({
                     { target: headerLayoutTitle }
                   )}
                   placeholder={formatMessage({
-                    id: 'app.component.search.placeholder',
+                    id: 'content-manager.components.Search.placeholder',
```
Comment (ronronscelestes) [inline packages/core/admin/admin/src/content-manager/pages/ListView/index.js:312]: actually the `content-manager.components.Search.placeholder` translation returns "Search for an entry..." 🤔
after discussing it with the designer, it should be changed by only "Search" without the dots!
Would you want to update it or I could handle it in another PR if you prefer 😊

**Review by ronronscelestes — CHANGES_REQUESTED** [review]
hi @yangfei4913438 thank you, nice catch 💃
only a small comment about the product copy!

File: packages/core/admin/admin/src/content-manager/pages/ListView/index.js (hunk 1 above)
Comment (yangfei4913438) [reply packages/core/admin/admin/src/content-manager/pages/ListView/index.js]: The translation text has been updated.

Comment (ronronscelestes) [reply packages/core/admin/admin/src/content-manager/pages/ListView/index.js]: thank you! 🙏

File: packages/core/admin/admin/src/translations/en.json [hunk 2]
```diff
@@ -679,6 +679,7 @@
   "content-manager.apiError.This attribute must be unique": "{field} must be unique",
   "form.button.continue": "Continue",
   "form.button.done": "Done",
+  "global.Search": "Search",
```
Comment (ronronscelestes) [inline packages/core/admin/admin/src/translations/en.json:682]: about using admin translations for plugins - after discussing it with the team, it sounds good indeed!

following up:
- could we use the same format as for others translations keys (all lowercase ➡️ `"global.search"`)?
- I think we could add this fix (#13533) into this PR, using the same key you've created. I'll close the other PR once this one is merged!

does that sound good to you?

**Comment by yangfei4913438** [comment]
merge into PR #13533

**Pull request closed without merging**